### PR TITLE
Updated may2015 doc

### PR DESCRIPTION
**Pull request by icon5585**

### CODE REVIEW

File: content/may2015/authorization.md [hunk 1]
```diff
@@ -1,45 +1,45 @@
 ---
-title: Authorization | FHIR MAY 2015 BALLOT API
+title: Authorization | FHIR DSTU 2 API
```
Comment (dennispatterson) [inline content/may2015/authorization.md]: Revert the version change

**Comment by jfrhong** [comment]
+1 after changing the version back

**Comment by icon5585** [comment]
Reverted the version back to may 2015 ballot, good catch!

**Comment by dennispatterson** [comment]
👍

**Pull request merged**